### ISSUE TITLE
[feature request] add shortcut to jump to matching bracket

### ISSUE THREAD

**Issue by iamqiz** [Tampermonkey/tampermonkey#1958]
(Please fill out the issue template with your details)

## Specifications

- Chromium: (i.e. 64.0.3282.140)
- TM: (i.e. 4.5.5648)
- OS: (i.e. MacOS 10.13.4)

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name MyName
// @namespace http://mywebsite.com
// @Version 1.0
// @description try to take over the world!
// @author Me
// @match http://mywebsite.com/*
// @require http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js
// @include https://mywebsite.com/*
// @grant none
// ==/UserScript==
```

**Comment by derjanb** [comment]
Should be fixed at 5.1.6193 ([crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|xpi in review)

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions` (after you've enabled 'Developer Mode').
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

**Comment by iamqiz** [comment]
@derjanb thank you very much for implementing the feature❤
just now I have used it, nice ~
by the way , can these shortcuts be changed in the settings?  thanks again
